packages/spark/base/inputs/_input-container.scss

```diff
@@ -11,3 +11,221 @@
 .sprk-b-InputContainer__visibility-toggle {
   margin: $sprk-visibility-control-margin;
 }
+
+// Applies to any huge input
+.sprk-b-InputContainer--huge .sprk-b-TextInput,
```

Comment:
Moved all of the huge input container styles into this input_container Sass file as they were previously inside label.scss and text-input.scss but now they are not only for text inputs but also selects.